src/spec-node/devContainersSpecCLI.ts

```diff
@@ -328,18 +328,24 @@ async function doBuild({
 			throw new ContainerError({ description: `Dev container config (${uriToFsPath(configFile || getDefaultDevContainerConfigPath(cliHost, workspace!.configFolderPath), cliHost.platform)}) not found.` });
 		}
 		const { config } = configs;
-		let imageNameResult = '';
+		let imageNameResult = [''];
+		
+		// Support multiple use of `--image-name` 
+		const imageNames = (argImageName && (Array.isArray(argImageName) ? argImageName : [argImageName]) as string[]) || undefined;
 
 		if (isDockerFileConfig(config)) {
 
 			// Build the base image and extend with features etc.
-			const { updatedImageName } = await buildNamedImageAndExtend(params, config, argImageName);
-
-			if (argImageName) {
-				if (!buildxPush) {
-					await dockerPtyCLI(params, 'tag', updatedImageName, argImageName);
+			let { updatedImageName } = await buildNamedImageAndExtend(params, config, imageNames);
+			updatedImageName = (updatedImageName && (Array.isArray(updatedImageName) ? updatedImageName : [updatedImageName]) as string[]) || [''];
+
+			if (imageNames) {
+				if (!buildxPush && typeof updatedImageName === 'string') {
+					imageNames.forEach(async function (image) {
```

Comment:
This wouldn't wait. Try something like:
await Promise.all(imageNames.map(imageName => dockerPtyCLI(params, 'tag', updatedImageName[0], imageName)));

Reply:
Ah! Modified., thanks